cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityUserOperatorTest.java

```diff
@@ -76,6 +76,7 @@
         expected.add(new EnvVarBuilder().withName(EntityUserOperator.ENV_VAR_ZOOKEEPER_SESSION_TIMEOUT_MS).withValue(String.valueOf(uoZookeeperSessionTimeout * 1000)).build());
         expected.add(new EnvVarBuilder().withName(EntityUserOperator.ENV_VAR_CLIENTS_CA_KEY_SECRET_NAME).withValue(KafkaCluster.clientsCaKeySecretName(cluster)).build());
         expected.add(new EnvVarBuilder().withName(EntityUserOperator.ENV_VAR_CLIENTS_CA_CERT_SECRET_NAME).withValue(KafkaCluster.clientsCaCertSecretName(cluster)).build());
+        expected.add(new EnvVarBuilder().withName(EntityUserOperator.ENV_VAR_CLIENTS_CA_NAMESPACE).withValue(namespace).build());
```

Comment:
Should we add a test dedicated to this where the watched namespace and the namespace of the Kafka resource differ?

Reply:
Actually, the current test has `uoWatchedNamespace` different from `namespace` since the beginning.

Reply:
Ok, great.In that case it should be all good.